deployments/apiv2/services/users/src/main.py

```diff
@@ -198,6 +222,17 @@ async def login(request: Request, details: UserLogin | CustomerLogin):
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+async def _update_failed_login_attempts(con, account_type: str, id: str, count: int) -> None:
+    if count == MAX_FAILED_LOGIN_ATTEMPTS:
+        # if max failed attempts is reached, then deactivate the account and increment count
+        update_query = f"UPDATE {account_type}s SET suspended='t', failed_login_attempts=failed_login_attempts+1 where id=$1"
```

Comment:
Do we need to suspend the user? I think this works fine, but just wondering if we want to also suspend the user

Reply:
I think so because then the admin can reactive them using the mechanisms we already have set up

Reply:
Good idea